docs/how-to/integrate_oai_ran_with_ue_simulator.md

```diff
@@ -0,0 +1,87 @@
+# Integrate Charmed OAI RAN with the OAI RAN UE Simulator
+
+Charmed OAI RAN includes a UE simulator that can be used to simulate 5G network traffic and validate the RAN deployment.
+
+This guide walks you through deploying the Charmed OAI RAN UE Simulator on a Kubernetes cluster using Juju and Terraform.
+
```

Comment:
An important step that seems to be missing here is that the `du` needs to be in `simulation-mode` through its configuration. I think a quick note here would be enough.

Reply:
Agreed, done